src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/ApiParameterDescriptionExtensions.cs

```diff
@@ -111,5 +112,17 @@ internal static bool IsFromForm(this ApiParameterDescription apiParameter)
             return (source == BindingSource.Form || source == BindingSource.FormFile)
                 || (elementType != null && typeof(IFormFile).IsAssignableFrom(elementType));
         }
+
+        internal static bool IsFromFormAttributeUsedWithFormFile(this ApiParameterDescription apiParameter)
+        {
+            // Retrieve parameter information
+            var parameterInfo = ParameterInfo(apiParameter);
+
+            // Retrieve attributes from the parameter
+            var fromFormAttribute = parameterInfo?.GetCustomAttribute<FromFormAttribute>();
+
+            return fromFormAttribute != null && parameterInfo?.ParameterType == typeof(IFormFile);
+
+        }
```

Comment:
As this is used in exactly one place, can you just move it to the other class, or are there other places where we're doing equivalent logic that could re-use this new method?

Reply:
I have moved it to the class from where I was calling it.